enum.c

```diff
@@ -558,20 +572,25 @@ reject_i(RB_BLOCK_CALL_FUNC_ARGLIST(i, ary))
 }
 
 /*
- *  call-seq:
- *     enum.reject { |obj| block } -> array
- *     enum.reject                 -> an_enumerator
+ * call-seq:
+ *   reject {|element| ... } -> array
+ *   reject -> enumerator
+ *
+ * Returns an array of objects determined by the block.
```

Comment:
`determined` -> `rejected` (similar to the language used for `select`).

Reply:
Done.